shared/gh/partials/admin-batch-edit-time-picker.html

```diff
@@ -1,15 +1,18 @@
+<% var dayIndex = parseInt(data.dayIndex, 10) %>
+<% var daysInUse = _.map(data.daysInUse, function(value, key) {return parseInt(key, 10)}) %>
```

Comment:
In terms of consistency, we might have to do:
`<% var daysInUse = _.map(data.daysInUse, function(value, key) { return parseInt(key, 10); }) %>`